core/model/src/main/java/org/eclipse/sw360/antenna/model/artifact/facts/java/MavenCoordinates.java

```diff
@@ -61,6 +61,17 @@ public String getVersion() {
         return version;
     }
 
+    @Override
+    public String getType() {
+        return "maven";
+    }
+
+    @Override
+    public String getPurl() {
```

Comment:
This specialization is not necessary with the generalization of namespaces, see above.

Reply:
I think, here we are in a more general discussion. If I add a getNamespace() method to ArtifactCoordinates, the whole concept of Artifact Coordinates could be simplified to one class as you did in ORT. But I did not want to open this box of Pandora for now. Making this overwrite is simplifying the world for all other coordinate types we have so far.